### DISCUSSION TITLE
Extracting Images from CloudPoints and GeoLocation

### DISCUSSION THREAD

**Discussion by joao-d-oliveira** [daavoo/pyntcloud#354] [Q&A]
I'll start by saying "it's a pleasure to meet you Cloud Point, heard a lot about you". 
I'm an ML Engineer, done a bunch of Computer Vision (CV) project, but this is the first with Cloud Point data... so I'm quite newbie to Cloud Point and would love to get some guidance.

In the field of Cloud Point my project consists into `geo-locating objects found by a CV AI Model `(and ideally measure them). 

Basically I broke down the problem into the simplified script:
1. Extracting images from the CloudPoint (with an easy way to geo-referencing them);
2. Run the images through AI Model (getting therefore the location of the object within the Image);
3. Convert those Image Locations into Cloud Point data.

Could you help me out guiding where to start, and how to achieve step 1) and 3) ?

Thanks!
 Joao
